### PR TITLE
GRAPHICS: Apply aspect ratio correction to 16 color VGA 640x350 video…

### PR DESCRIPTION
**Pull request by yuv422**
Darkseed uses an unusual video mode 640x350 16 colors. This should be aspect ratio corrected.
This PR applies the Aspect Ratio Correction setting to this video mode.

scummvm-darkseed-cd-00013

### CODE REVIEW

**Comment by yuv422** [comment]
I noticed that there are a lot of implementations of `gameNeedsAspectRatioCorrection()` I assume that I only need to add it to this one as the others are for 3d games?

**Comment by ccawley2011** [comment]
Do you have any screenshots with aspect ratio correction disabled for comparison?

File: backends/graphics/opengl/opengl-graphics.cpp [hunk 1]
```diff
@@ -1633,9 +1633,10 @@ bool OpenGLGraphicsManager::gameNeedsAspectRatioCorrection() const {
 		const uint height = getHeight();
 
 		// In case we enable aspect ratio correction we force a 4/3 ratio.
-		// But just for 320x200, 640x400 and Hercules games, since other
+		// But just for 320x200, 640x400, 640x350 (16 color VGA) and Hercules games, since other
```
Comment (NMIError) [inline backends/graphics/opengl/opengl-graphics.cpp:1636]: Nitpick: 640 x 350 is an EGA resolution. Dark Seed uses a trick to use the VGA palette in this mode, but that is not normally available.

**Comment by yuv422** [comment]
> Do you have any screenshots with aspect ratio correction disabled for comparison?

Yeah it looks like this when uncorrected.

scummvm-darkseed-cd-00019

File: backends/graphics/opengl/opengl-graphics.cpp (hunk 1 above)
Comment (yuv422) [reply backends/graphics/opengl/opengl-graphics.cpp:1636]: Good point I'll change the text to clarify that

**Comment by bluegr** [comment]
Thanks!

**Pull request merged**